common/lib/common-definitions/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # @fluidframework/common-definitions Changelog
 
-## [1.2.0](https://github.com/microsoft/FluidFramework/releases/tag/common-definitions_v1.2.0)
+## [1.0.0](https://github.com/microsoft/FluidFramework/releases/tag/common-definitions_v1.0.0)
```

Comment:
Why did this version get lowered?

Reply:
It was committed at the wrong version. I checked the release history and we have not released 1.0 of common refs. I'll double check before merge.